lib/models/auth.dart

```diff
@@ -631,6 +631,10 @@ class AuthModel with ChangeNotifier {
       _accounts = (json.decode(str ?? '[]') as List)
           .map((item) => Account.fromJson(item))
           .toList();
+      activeAccountIndex = prefs.getInt(StorageKeys.defaultAccount);
+      _activeTab = prefs.getInt(
+              StorageKeys.getDefaultStartTabKey(activeAccount.platform)) ??
+          0;
```

Comment:
How would it behave if there is no account in the list?

Reply:
It opens the login screen. [This](https://github.com/git-touch/git-touch/blob/a1021465a8c5b1ca5d92063a27a1d9d077013cd7/lib/home.dart#L217-L219) handles that case.